backend/app/core/conf.py

```diff
@@ -75,6 +84,9 @@ class Settings(BaseSettings):
     MIDDLEWARE_GZIP: bool = True
     MIDDLEWARE_ACCESS: bool = False
 
+    class Config:
+        env_file = '.env'
+
```

Comment:
It might be possible to be more flexible here and specify that the core directory holds

E.g.:
```python
__RootDir = Path(__file__).resolve().parent.parent.parent
__Env_Dev: str = os.path.join(__RootDir, 'app', 'core', 'dev.env')
__Env_Pro: str = os.path.join(__RootDir, 'app', 'core', 'pro.env')
__Use_Env: str = __Env_Dev if ENVIRONMENT == 'dev' else __Use_Pro

class Config:
    env_file = __Use_Env
```
You can place the private constants outside the Settings class to make it look better

Reply:
In fact pydantic has considered some situations

![image](https://user-images.githubusercontent.com/41266749/233236252-4b22904b-2b15-4134-9cdc-33baa1933caa.png)

Python package import has always been a troublesome problem for novices, so I will avoid using paths as much as possible

Reply:
Agree